pkgs/by-name/li/libblake3/package.nix

```diff
@@ -22,6 +22,8 @@ stdenv.mkDerivation (finalAttrs: {
     cmake
   ];
 
+  cmakeFlags = lib.optional (!stdenv.hostPlatform.isStatic) "-DBUILD_SHARED_LIBS=ON";
```

Comment:
We could set the option on static builds as well, so we don't depend on the default:

```suggestion
  cmakeFlags = [
    (lib.cmakeBool "BUILD_SHARED_LIBS" (!stdenv.hostPlatform.isStatic))
  ];
```

Reply:
Thanks, I updated the PR with you suggestion.